File: FrontEnd/src/app/features/billing/receiptsViewer/classes/view-models/receipt-pdf-vm.ts
import { ReceiptPdfBankAccountVM } from './receipt-pdf-bankAccount-vm'
import { ReceiptPdfHeaderVM } from './receipt-pdf-header-vm'
import { ReceiptPdfPartyTypeVM } from './receipt-pdf-partyType-vm'

export interface ReceiptPdfVM {

    header: ReceiptPdfHeaderVM
    issuer: ReceiptPdfPartyTypeVM
    customer: ReceiptPdfPartyTypeVM
    paymentMethod: string
    bankAccounts: ReceiptPdfBankAccountVM[]
    amount: number
    previousBalance: number
    newBalance: number

}

File: API/Features/Billing/Receipts/ViewModels/Pdf/ReceiptPdfVM.cs
using System;
using API.Infrastructure.Classes;

namespace API.Features.Billing.Receipts {

    public class ReceiptPdfVM {

        public Guid InvoiceId { get; set; }
        public InvoicePdfHeaderVM Header { get; set; }
        public string Remarks { get; set; }
        public InvoicePdfPartyVM Customer { get; set; }
        public InvoicePdfDocumentTypeVM DocumentType { get; set; }
        public string PaymentMethod { get; set; }
        public InvoicePdfPartyVM Issuer { get; set; }
        public InvoicePdfSummaryVM Summary { get; set; }
        public decimal PreviousBalance { get; set; }
        public decimal NewBalance { get; set; }
        public SimpleEntity[] BankAccounts { get; set; }

    }

}